extensions/classifiers/PythonProgramTokenizer.js

```diff
@@ -333,31 +322,27 @@ oppia.factory('PythonProgramTokenizer', [
                   needcont = 1;
                   contline = line;
                   break;
-                }
-                else {
+                } else {
                   tokenizedProgram.push(
                     [PythonProgramTokenType.STRING, token]);
                 }
-              }
+              } // eslint-disable-line brace-style
               // Ordinary name
               else if (namechars.indexOf(initial) !== -1) {
                 tokenizedProgram.push([PythonProgramTokenType.NAME, token]);
-              }
+              } // eslint-disable-line brace-style
```

Comment:
Ditto here and elsewhere. (Please also change stmt to statement while you're at it.)

Reply:
Done, thanks!